src/slider/slick/dots.jsx

```diff
@@ -66,18 +81,25 @@ export default class Dots extends React.Component {
             }
 
             children.push(
-                <li key={i} className={itemClazz} {...handleProp}>
-                    {
-                        (dotsRender instanceof Function) ?
-                            <span>
-                                {dotsRender(docIndex, currentSlideIndex)}
-                            </span> :
-                            <button />
-                    }
+                <li
+                    key={i}
+                    className={itemClazz}
+                    role="listitem"
+                    {...handleProp}
+                >
+                    {dotsRender instanceof Function ? (
+                        <span>{dotsRender(docIndex, currentSlideIndex)}</span>
+                    ) : (
+                        <button tabIndex="-1" />
```

Comment:
When using a discouraged practice, like setting `tabIndex="-1"` its helpful to have a comment why. Will help the next maintainer. 
Suggested comment: `Slider is navigated by right and left arrow buttons so the dots are not required functionality`

Reply:
OK~